### PR TITLE
Add Accordion component 

### LINKED ISSUE #8: Add accordion component

### PR DESCRIPTION
**Pull request by JuKahl**
- 🔨 Add accordion component
- 💄 Add styling
- ♻️ Refactor naming

[Deployment](https://king-of-the-accordion-yxqoffla.herokuapp.com/storybook/index.html?path=/story/common-accordion--primary)

### CODE REVIEW

File: components/accordion/Accordion.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React, { useState } from "react";
+import styles from "./Accordion.module.css";
+
+export type Content = {
```
Comment (fre-ben) [inline components/accordion/Accordion.tsx]: You propably should rename this Prop type for better understanding later on :)

```suggestion
export type AccordionProps = {
```

**Review by fre-ben — APPROVED** [review]
LGTM 🧛 
Would only suggest one small naming change

**Review by mhstrkmp — COMMENTED** [review]
lgtm 👍

**Review by SilviaBarcellini — APPROVED** [review]
lgtm 👍

**Review by RonjaMeinecke — APPROVED** [review]
🦸🏽‍♀️🦾stark

**Review by ghost — APPROVED** [review]
LGTM 🚀

**Pull request merged**